source/css/db-ui-base.scss

```diff
@@ -37,3 +36,9 @@ h6 {
 main {
 	padding: $db-spacing-1 $db-spacing-3;
 }
+
+.typography-table {
+	tr > td {
+		padding: $db-spacing-0-5 $db-spacing-1;
```

Comment:
`$db-spacing-1` doesn't seem to be defined, as this is the output / error on the console on `build`.

Reply:
I used fixed px for now because we will change the spacing system anyways